### ISSUE TITLE
Get error when try to get tollbooth_chi

### ISSUE THREAD

**Issue by BrightRan** [didip/tollbooth_chi#6]
When using the following command to try to get **`tollbooth_chi`**,
```go
go get -u "github.com/didip/tollbooth_chi"
```
it always returns the "**cannot find package**" error like as this:
```go
cannot find package "github.com/didip/tollbooth/v6" in any of:
	/opt/hostedtoolcache/go/1.15.0/x64/src/github.com/didip/tollbooth/v6 (from $GOROOT)
	/home/runner/go/src/github.com/didip/tollbooth/v6 (from $GOPATH)
cannot find package "github.com/didip/tollbooth/v6/limiter" in any of:
	/opt/hostedtoolcache/go/1.15.0/x64/src/github.com/didip/tollbooth/v6/limiter (from $GOROOT)
	/home/runner/go/src/github.com/didip/tollbooth/v6/limiter (from $GOPATH)
```
I tested on my local environment and in a GitHub Actions workflow, both get the same error.
* On my local environment (**Go v1.15.0** installed).
![error](https://user-images.githubusercontent.com/46307996/91517822-5657ab00-e921-11ea-827c-24ba699586f0.PNG)

* In the GitHub Actions workflow (see [here](https://github.com/TestWorkflowsRML/TestGo/runs/1039696413?check_suite_focus=true)).
```yaml
jobs:
  build:
    name: Build
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
    steps:
      - name: Setup Go
        uses: actions/setup-go@v2.1.2
        with:
          go-version: 1.15.0

      - name: Install dependencies
        shell: bash
        run: go get -u "github.com/didip/tollbooth_chi"
```

**Comment by dineshkumarj** [comment]
@BrightRan - Thanks for raising this ticket. I am able to reproduce this issue on my local machine after the deleting the package tollbooth and tollbooth_chi from the respective directory.

Scenario 1 - I have installed **tollbooth** successfully, then when I try to install **tollbooth_chi** it throws me an error "cannot find package",
<img width="639" alt="Screenshot 2020-08-28 at 11 56 01 AM" src="https://user-images.githubusercontent.com/12094640/91528534-8bfe9300-e925-11ea-8681-bb955f84eb46.png">

However, I could see the tollbooth_chi created in the respective directory. But still my test case fails,
<img width="1114" alt="Screenshot 2020-08-28 at 1 27 27 PM" src="https://user-images.githubusercontent.com/12094640/91536370-3ed4ee00-e932-11ea-9ae7-2d514a32552e.png">

Scenario 2 - I have installed **tollbooth** successfully, then I browsed to the directory where this is installed as per the below image and I tried to install **tollbooth_chi**, it shows like it is installed but could see the respective tollbooth_chi directory nor I was able to run my script which produces the same error - cannot find package,
<img width="1114" alt="Screenshot 2020-08-28 at 1 21 40 PM" src="https://user-images.githubusercontent.com/12094640/91535855-6f685800-e931-11ea-9a79-4fcaf61e0837.png">
